platform/app/src/utils/isSeriesFilterUsed.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This function is used to check if the filter is used. Its intend is to
+ * warn the user in case of link with a SeriesInstanceUID was called
+ * @param instances
+ * @returns
+ */
+export default function isSeriesFilterUsed(instances, filters) {
+  const seriesInstanceUIDs = filters?.seriesInstanceUID;
+  if (seriesInstanceUIDs) {
```

Comment:
I am a bit confused by the logic of this function. If I am not mistaken, this check on line 9 and subsequent return on line 10 means that when there ARE seriesInstanceUIDs line 15 will NOT get executed. Furthermore when seriesInstanceUIDs is null or undefined then line 15 will cause an error. Furthermore, the JS Doc comment for this function is confusing too.

Lastly I tried this test URL which I thought might show the notification warning/error but does NOT!

https://deploy-preview-3592--ohif-dev.netlify.app/viewer?StudyInstanceUIDs=1.3.6.1.4.1.14519.5.2.1.256467663913010332776401703474716742458&SeriesInstanceUID=blah.blah

Reply:
You are conpletely right. When I refactor the function, I forgot the !